Mockups/MockedRun.debug.cs

```diff
@@ -23,7 +23,7 @@ protected MockedRun()
         /// <summary>
         /// Determines whether this run has been initialized. 
         /// </summary>
-        public bool IsInitialized { get; private set; }
+        public bool IsInitialized { get; set; }
```

Comment:
It's been a while, but I have a feeling this is wrong. This should probably not be settable from the outside.